etcdserver/etcdserverpb/raft_internal.proto

```diff
@@ -42,8 +42,9 @@ message InternalRaftRequest {
   AuthRoleAddRequest auth_role_add = 16;
   AuthRoleGrantRequest auth_role_grant = 17;
   AuthenticateRequest authenticate = 18;
+  AuthUserGetRequest auth_user_get = 19;
 
-  AlarmRequest alarm = 19;
+  AlarmRequest alarm = 20;
```

Comment:
let move alarmRequest to number 10?

Also let give all auth req/resp a higher number? say start from 1000? 

For etcd3 beta, we can still modify anything after field number 9. After we release 3, it would be hard for us to modify these. I feel eventually, we need to have more stuff in auth.